src/support/azext_support/aaz/latest/support/in_subscription/file/_list.py

```diff
@@ -0,0 +1,219 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "support in-subscription file list",
+    is_preview=True,
+)
+class List(AAZCommand):
+    """List all the Files information under a workspace for an Azure subscription.
+
+    :example: List all the files within a file-workspace for a specified subscription
+        az support in-subscription file-workspace file list --file-workspace-name "TestFileWorkspaceName" --subscription "TestSubscription"
```

Comment:
We dont need to provide subscription in the help command, it takes the default  subscription from az login